tests/Microsoft.Identity.Test.Core.UIAutomation/CoreUiTestConstants.cs

```diff
@@ -33,66 +33,59 @@ public static class CoreUiTestConstants
         public const string TestResultFailureMessage = "Result: Failure";
+        public const string ClearAllCacheId = "ClearAllCache";
 
-        //MSAL test app
         public const string DefaultScope = "User.Read";
-        public const string AcquirePageID = "Acquire";
-        public const string CachePageID = "Cache";
-        public const string SettingsPageID = "Settings";
-        public const string ScopesEntryID = "scopesList";
-        public const string UiBehaviorPickerID = "uiBehavior";
+        public const string AcquirePageId = "Acquire";
+        public const string CachePageId = "Cache";
+        public const string SettingsPageId = "Settings";
+        public const string ScopesEntryId = "scopesList";
+        public const string UiBehaviorPickerId = "uiBehavior";
         public const string SelectUser = "userList";
         public const string UserNotSelected = "not selected";
         public const string UserMissingFromResponse = "Missing from the token response";
         public const string RedirectUriOnAndroid = "urn:ietf:wg:oauth:2.0:oob";
-        public const string RedirectUriEntryID = "redirectUriEntry";
+        public const string RedirectUriEntryId = "redirectUriEntry";
 
         //MSAL B2C
-        public const string B2cScopes = "https://sometenant.onmicrosoft.com/some/scope";
-        public const string AuthorityPickerID = "b2cAuthorityPicker";
-        public const string WebUPNB2CLocalInputID = "logonIdentifier";
-        public const string B2CWebSubmitID = "next";
-        public const string B2CWebPasswordID = "password";
+        public const string AuthorityPickerId = "b2cAuthorityPicker";
+        public const string WebUpnB2CLocalInputId = "logonIdentifier";
+        public const string B2CWebSubmitId = "next";
+        public const string B2CWebPasswordId = "password";
         public const string B2CLoginAuthority = "b2clogin.com";
         public const string MicrosoftOnlineAuthority = "login.microsoftonline.com";
         public const string NonB2CAuthority = "non-b2c authority";
         public const string B2CEditProfileAuthority = "Edit profile policy authority";
-        public const string FacebookAccountID = "FacebookExchange";
-        public const string WebUPNB2CFacebookInputID = "m_login_email";
-        public const string B2CWebPasswordFacebookID = "m_login_password";
-        public const string B2CFacebookSubmitID = "u_0_5";
-        public const string GoogleAccountID = "GoogleExchange";
-        public const string WebUPNB2CGoogleInputID = "Email";
-        public const string B2CWebPasswordGoogleID = "Passwd";
-        public const string B2CGoogleNextID = "next";
-        public const string B2CGoogleSignInID = "signIn";
-        public const string B2CEditProfileContinueID = "continue";
+        public const string FacebookAccountId = "FacebookExchange";
+        public const string WebUpnB2CFacebookInputId = "m_login_email";
```

Comment:
#TheHeroWeDeserve  (thanks for cleaning these up)

Reply:
:sunglasses: 

---
In reply to: [246214528](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/762#discussion_r246214528) [](ancestors = 246214528)